packages/documentation/src/stories/components/switch/switch.snapshot.stories.ts

```diff
@@ -0,0 +1,41 @@
+import type { Args, StoryContext, StoryObj } from '@storybook/web-components';
+import meta, { Default } from './switch.stories';
+import { html } from 'lit';
+import { bombArgs } from '../../../utils/bombArgs';
+
+export default {
+  ...meta,
+  title: 'Snapshots',
+};
+
+type Story = StoryObj;
+
+export const Switch: Story = {
+  render: (_args: Args, context: StoryContext) => {
+    const templateVariants = bombArgs({
+      labelPosition: ['both', 'before', 'after'],
+      labelBefore: ['Off', 'this is longer text'],
+      labelAfter: ['On', 'this is longer text'],
+      hiddenLabel: [false],
+      checked: [false, true],
+      disabled: [false, true],
+      validation: ['null', 'is-valid', 'is-invalid'],
+    }).map((args: Args) => {
```

Comment:
Maybe you could add a filter that filters out all variations with altered label text except one where both for one or three if you want all combinations.